src/handlers.rs

```diff
@@ -423,7 +427,7 @@ pub(crate) async fn handle_online_wallet_subcommand(
             Ok(json!({}))
         }
         Sync => {
-            #[cfg(any(feature = "electrum", feature = "esplora"))]
+            #[cfg(any(feature = "electrum", feature = "esplora", feature = "cbf"))]
```

Comment:
This line doesn't need the `cbf` feature gate since `request` isn't used by kyoto and having it causes a unused warning during build if only the `cbf` feature is enabled.
```suggestion
            #[cfg(any(feature = "electrum", feature = "esplora"))]
```

Reply:
fixed.